api/all.yaml

```diff
@@ -722,6 +748,268 @@ paths:
           content:
             application/json:
               schema: {}
+  /models:
+    post:
+      operationId: createModel
+      summary: |
+        Create a model.
+        WARNING: This API is experimental and will change in future versions.
+      tags:
+        - Models
+      description: |
+        Creates a new model instance.
+        WARNING: This API is experimental and will change in future versions.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateModel'
+      responses:
+        '200':
+          description: The new model was successfully created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ModelInfo'
+    get:
+      tags:
+        - Models
+      parameters:
+        - name: catalog_name
+          in: query
+          description: Name of parent catalog for models of interest.
+          required: true
+          schema:
+            type: string
+        - name: schema_name
+          in: query
+          description: Parent schema of model.
+          required: true
+          schema:
+            type: string
+        - name: max_results
+          in: query
+          description: |
+            Maximum number of models to return.
+            - when set to a value greater than 0, the page length is the minimum of this value and a server configured value;
+            - when set to 0, the page length is set to a server configured value;
+            - when set to a value less than 0, an invalid parameter error is returned;
```

Comment:
Does the server configured value apply to UC OSS as well (or it's a databricks-only thing)?

Reply:
This was copied from another paginated api.  We can update this when the other pagination descriptions are updated.